spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/keras/SeparableConvolution2D.scala

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2016 The BigDL Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.intel.analytics.bigdl.nn.keras
+
+import com.intel.analytics.bigdl.nn.SpatialSeperableConvolution
+import com.intel.analytics.bigdl.nn.abstractnn._
+import com.intel.analytics.bigdl.optim.Regularizer
+import com.intel.analytics.bigdl.tensor.Tensor
+import com.intel.analytics.bigdl.tensor.TensorNumericMath.TensorNumeric
+import com.intel.analytics.bigdl.utils.Shape
+import com.intel.analytics.bigdl.nn.abstractnn.{AbstractModule, DataFormat}
+
+import scala.reflect.ClassTag
+
+class SeparableConvolution2D[T: ClassTag](val nbFilter: Int,
+                                          val nbCol: Int,
+                                          val nbRow: Int,
+                                          val depthMultiplier: Int = 1,
+                                          val subsample: (Int, Int) = (1, 1),
+                                          val borderMode: String = "valid",
+                                          val bias: Boolean = true,
+                                          val format: DataFormat = DataFormat.NCHW,
```

Comment:
Missing activation?
Also add init, refer to Dense

Reply:
Please refer to : https://faroit.github.io/keras-docs/1.2.2/layers/convolutional/#separableconvolution2d for those parameters